CHANGELOG.md

```diff
@@ -1,5 +1,60 @@
 # Cromwell Change Log
 
+## 28
+
+### Call Caching
+
+* New endpoint returning the hash differential for 2 calls. 
+
+`GET /api/workflows/:version/callcaching/diff`
+
+The endpoint returns a JSON response such as the following:
+
+```json
+{
+    "callA": {
+        "workflowId": "85174842-4a44-4355-a3a9-3a711ce556f1",
+        "callFqn": "wf_hello.hello",
+        "jobIndex": -1,
+        "allowResultReuse": true
+    },
+    "callB": {
+        "workflowId": "7479f8a8-efa4-46e4-af0d-802addc66e5d",
+        "callFqn": "wf_hello.hello",
+        "jobIndex": -1,
+        "allowResultReuse": true
+    },
+    "hashDifferential": [
+        {
+            "command template": {
```

Comment:
Yeah, unless you have a rainbow table of MD5s for common integer values, they're essentially meaningless.

Except... they *do* look authoritative: 
> Oh yeah, the hashes *are* different, see, that one starts with `a5` and that one starts with `b7`...